packages/url_launcher/url_launcher/test/url_launcher_test.dart

```diff
@@ -281,6 +281,42 @@ void main() {
       await launchResult;
       expect(binding.renderView.automaticSystemUiAdjustment, true);
     });
+
+    test('open remote desktop url', () async {
```

Comment:
Let's call this 'open non-parseable URL' so it's clear what the real purpose of this test is.